contracts/0.8.9/utils/PausableUntil.sol

```diff
@@ -43,10 +43,6 @@ contract PausableUntil {
         return block.timestamp < RESUME_SINCE_TIMESTAMP_POSITION.getStorageUint256();
     }
 
-    function _initializePausable() internal {
```

Comment:
Since we are trying to modify this contract, I'd like to ask about an additional change:
can we expose the state of the pause slot value with a getter?

Reply:
Added. Also added tests for the contract.